File: btjs_page/btjs/mod/checkers/is_byte_256.js




export function is_byte256(data) {
    if( ! Number.isInteger(data)) {
        return false;
    }
    if(data >= 256) { return false; }
    if(data < 0) { return false; }
    return true;
}

export function is_byte127(data) {
    if( ! Number.isInteger(data)) {
        return false;
    }
    if(data >= 127) { return false; }
    if(data <= -127) { return false; }
    return true;
}

File: btpy/src/btpy/btpy_checkers/mod/is_byte_256/is_byte_256.py




def is_byte_256(ANY_DATA : any)-> bool:
    """
    TESTED
    Function that returns true if 
    the data sent is a byte 256 type, 
    that is, an 8-bit integer in the 
    range between 0 and 256.
    """
    if(not type(ANY_DATA) == int): 
        return False
    if(ANY_DATA >= 256): return False
    if(ANY_DATA < 0): return False
    return True